LDAP integration test panics

```
2015/08/09 16:32:21 Client [2] sent abandon signal to request[messageID = 1]
panic: close of nil channel
goroutine 1589 [running]:
github.com/vjeantet/ldapserver.(*Message).Abandon(0xc208c0e140)
	/home/travis/gopath/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/vjeantet/ldapserver/message.go:54 +0x2c
created by github.com/vjeantet/ldapserver.(*client).close
	/home/travis/gopath/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/vjeantet/ldapserver/client.go:166 +0x3e0
```

from https://travis-ci.org/openshift/origin/jobs/74807522

Saw this again in https://travis-ci.org/openshift/origin/jobs/75549066

```
2015/08/14 05:09:27 Error readMessagePacket: EOF
2015/08/14 05:09:27 client 2 close()
2015/08/14 05:09:27 Client [2] sent abandon signal to request[messageID = 1]
panic: close of nil channel
goroutine 1607 [running]:
github.com/vjeantet/ldapserver.(*Message).Abandon(0xc20a5a3bc0)
	/home/travis/gopath/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/vjeantet/ldapserver/message.go:54 +0x2c
created by github.com/vjeantet/ldapserver.(*client).close
	/home/travis/gopath/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/vjeantet/ldapserver/client.go:166 +0x3e0
goroutine 1 [chan receive]:
testing.RunTests(0x2532650, 0x3009480, 0x47, 0x47, 0x1)
	/usr/local/go/src/testing/testing.go:556 +0xad6
testing.(*M).Run(0xc20836c460, 0x38e12c0)
	/usr/local/go/src/testing/testing.go:485 +0x6c
main.main()
	github.com/openshift/origin/test/integration/_test/_testmain.go:192 +0x1d5
```

--- a/Godeps/Godeps.json
+++ b/Godeps/Godeps.json
@@ -786,6 +786,7 @@
 		},
 		{
 			"ImportPath": "github.com/vjeantet/ldapserver",
+			"Comment": "UPSTREAM pull requests: 15",
 			"Rev": "5700661e721f508db936af42597a254c4ea6aea4"
 		},
 		{

--- a/Godeps/_workspace/src/github.com/vjeantet/ldapserver/message.go
+++ b/Godeps/_workspace/src/github.com/vjeantet/ldapserver/message.go
@@ -51,7 +51,9 @@ func (m *Message) String() string {
 // Abandon close the Done channel, to notify handler's user function to stop any
 // running process
 func (m *Message) Abandon() {
-	close(m.Done)
+	if m.Done != nil {
+		close(m.Done)
+	}
 }
 
 //GetDoneChannel return a channel, which indicate the the request should be

